closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -129,33 +160,24 @@ public static final boolean isAlgorithmEnabled(String property, String name, boo
         }
         if (useNativeAlgorithm) {
             /*
-             * User wants to use the native crypto implementation. Ensure that the
-             * native crypto library is loaded successfully. Otherwise, issue a warning
-             * message and fall back to the built-in java crypto implementation.
+             * User wants to use the native crypto implementation. Ensure that the native crypto library is enabled
+             * and the additional requirements are met. Otherwise, issue a warning message.
              */
-            if (isLoaded()) {
-                if (satisfied) {
-                    if (traceEnabled) {
-                        System.err.println(name + " - using native crypto library.");
-                    }
-                } else {
-                    useNativeAlgorithm = false;
-                    if (traceEnabled) {
-                        System.err.println("Warning: " + name + " native requirements not satisfied. " +
-                                explanation + " Using Java crypto implementation.");
-                    }
+            if (satisfied) {
+                if (traceEnabled) {
+                    System.err.println(name + " native crypto implementation enabled.");
                 }
             } else {
                 useNativeAlgorithm = false;
                 if (traceEnabled) {
-                    System.err.println("Warning: Native crypto library load failed." +
-                            " Using Java crypto implementation.");
+                    System.err.println("Warning: " + name + " native requirements not satisfied. " +
+                        explanation + " Using Java crypto implementation.");
```

Comment:
Please indent continued lines 8 spaces (like it was before).

Reply:
Updated